xml/common_intro_lifecycle_support_i.xml

```diff
@@ -0,0 +1,185 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 version="5.0" xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>Product Life Cycle and Support</title>
+
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker>
+<!--<dm:assignee></dm:assignee>-->
+   </dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  &suse; products are supported for up to 13 years. To check the life cycle
+  dates for your product, see
+  <link
+   xlink:href="https://www.suse.com/lifecycle/"/>.
+ </para>
+
+ <para>
+  For &sle; the following life cycles and release cycles apply:
+ </para>
+
+ <itemizedlist>
+  <listitem>
+   <para>
+    &sls; has a 13-year life cycle: 10 years of general support and three years
+    of extended support.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    &sled; has a 10-year life cycle: seven years of general support and three
+    years of extended support.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Major releases are published every four years. Service packs are published
+    every 12-14 months.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    &suse; supports previous &sle; service packs for six months after the
+    release of a new service pack.
+   </para>
+  </listitem>
+ </itemizedlist>
+
+ <para>
+  For some products, Long Term Service Pack Support (LTSS) is available. Find
+  information about our support policy and options at
+  <link
+   xlink:href="https://www.suse.com/support/policy.html"/> and
+  <link
+   xlink:href="https://www.suse.com/support/programs/long-term-service-pack-support.html"
+  />.
+ </para>
+
+ <para os="sles;sled">
+  In contrast to products, individual modules have a different life cycle and
```

Comment:
Fixed the sentence in question, thanks. About the profiling: Right, in this context, it's not needed, therefore I removed it (for doc-kit, I have to strip down the text anyway so it can be applied to multiple products, not only SLE).